LCM/scripts/python3/StatusReport.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/sh
+
+# $0 JobId (StartTime|EndTime) [ErrorMessage [SourceInfo ModuleName DurationInSeconds InstanceName StartDate ResourceName ModuleVersion RebootRequested ResourceId ConfigurationName InDesiredState]]
+
+curDate=`date +%Y-%m-%dT%T.%N%:z`
+ErrorMessage=""
+Errors=""
+RNIDS=""
+
+if [ "$2" = "EndTime" ]; then
```

Comment:
This would be a bigger effort, plus would be required to be made for all existing scripts as well, not just python 3.  We should have a different pr for this.